modules/spring-config/src/test/java/org/mule/config/spring/parsers/processors/CheckRequiredAttributesTestCase.java

```diff
@@ -223,7 +223,7 @@ public void testRealWorld() throws ParserConfigurationException
         assertOk(groups, "from id name type");
     }
 
-    protected PreProcessor createCheck(String[][] constraint)
+    protected PreProcessor createCheck(String[][] constraint, String elementName, String elementNamespaceUrl)
```

Comment:
is this a method override? If yes, then add @Override... Otherwise why are you sending two attributes you don't use?